### PR TITLE
simplify if statement

### PR DESCRIPTION
**Pull request by foxyriver**
**What this PR does / why we need it**:

simplify if statement

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: pkg/kubelet/kubelet_cadvisor.go [hunk 1]
```diff
@@ -78,15 +78,15 @@ func (kl *Kubelet) RootFsInfo() (cadvisorapiv2.FsInfo, error) {
...
-		}, nil
 	}
+	containerInfo, err := kl.cadvisor.ContainerInfo(containerName, req)
+	if err != nil {
+		return nil, err
+	}
+	return map[string]*cadvisorapi.ContainerInfo{
+		containerInfo.Name: containerInfo,
+	}, nil
+
```
Comment (dashpole) [inline pkg/kubelet/kubelet_cadvisor.go]: nit: remove extra empty line

**Comment by dashpole** [comment]
@k8s-bot ok to test

**Comment by foxyriver** [comment]
@dashpole have removed extra empty line, PTAL

**Comment by dashpole** [comment]
@foxyriver These changes /lgtm .  Our tests can be somewhat flaky ATM, so we may have to retry some of them.

**Pull request closed without merging**